VenafiPS/Public/Get-VenafiIdentity.ps1

```diff
@@ -0,0 +1,250 @@
+<#
+.SYNOPSIS
+Get user and group details
+
+.DESCRIPTION
+Returns user/group information for VaaS and TPP.
+For VaaS, this returns user information.
+For TPP, this returns individual identity, group identity, or distribution groups from a local or non-local provider such as Active Directory.
+
+.PARAMETER ID
+For TPP this is the guid or prefixed universal id.  To search, use Find-TppIdentity.
+For VaaS this can either be the user id (guid) or username which is the email address.
+
+.PARAMETER IncludeAssociated
+Include all associated identity groups and folders.  TPP only.
+
+.PARAMETER IncludeMembers
+Include all individual members if the ID is a group.  TPP only.
```

Comment:
Are the `-IncludeAssociated` and `-IncludeMembers` specific to local TPP Teams? If so, this should be documented.

Reply:
It's documented as 'TPP only' above.  I've also added a warning message if those params are used with vaas.